src/libraries/System.Drawing.Common/tests/runtimeconfig.template.json

```diff
@@ -0,0 +1,5 @@
+{
+    "configProperties": {
+        "System.Drawing.EnableUnixSupport": true
```

Comment:
Can we add a test project that doesn't have this set and validate we get the expected behavior?

Reply:
Rather than adding a new test project, is it OK to just add a test that uses remote executor to set the app context switch and validate the exception is thrown?

Reply:
Yup

Reply:
Done, thanks.